README.md

```diff
@@ -87,13 +157,28 @@ dd if=target/x86_64-bootloader/release/bootloader.bin of=/dev/sdX && sync
 
 Where sdX is the device name of your USB stick. **Be careful** to choose the correct device name, because everything on that device is overwritten.
 
+## Debugging
+Set a breakpoint at address `0x7c00`. Disassemble instructions with gdb:
```

Comment:
This address will differ between versions of bootloader and possibly between assembler versions.

Reply:
That's the address BIOS jumps to to start code execution, we'll always have *something* there

Reply:
Ah, ok.